doc/en/reference/reference.rst

```diff
@@ -246,7 +262,7 @@ Marks a test function as *expected to fail*.
         to specify ``reason`` (see :ref:`condition string <string conditions>`).
     :keyword str reason:
         Reason why the test function is marked as xfail.
-    :keyword Type[Exception] raises:
+    :keyword Type raises:
```

Comment:
Unfortunately `Type[Exception]` became ambiguous because of the new `Exception` classes defined in this file:

```
E:\projects\pytest\doc\en\reference\reference.rst:257: WARNING: more than one target found for cross reference 'Exception': pytest.fail.Exception, pytest.skip.Exception, pytest.xfail.Exception, pytest.exit.Exception
```

I tried to use `Type[builtins.Exception]` but that did not work.

I think removing `Exception` here is OK compared to the added documentation for the other functions, but suggestions are welcome.

Reply:
```diff
diff --git a/doc/en/reference/reference.rst b/doc/en/reference/reference.rst
index aa4c61e21..e46478da0 100644
--- a/doc/en/reference/reference.rst
+++ b/doc/en/reference/reference.rst
@@ -262,9 +262,10 @@ Marks a test function as *expected to fail*.
         to specify ``reason`` (see :ref:`condition string <string conditions>`).
     :keyword str reason:
         Reason why the test function is marked as xfail.
-    :keyword Type raises:
+    :keyword raises:
         Exception class (or tuple of classes) expected to be raised by the test function; other exceptions will fail the test.
         Note that subclasses of the classes passed will also result in a match (similar to how the ``except`` statement works).
+    :type raises: Type[:py:exc:`Exception`]
 
     :keyword bool run:
         Whether the test function should actually be executed. If ``False``, the function will always xfail and will

```

Seems to do the job. Weirdly, it's rendered slightly differently:

![image](https://github.com/pytest-dev/pytest/assets/625793/562f9b3b-2d10-46ab-ba6f-19de82e2e0f5)

![image](https://github.com/pytest-dev/pytest/assets/625793/51b0dfc8-cd34-4cab-8aaf-7db052c7d0e1)

but I'd say that doesn't really matter much.

Reply:
Done, thanks!